src/modules/job-manager/alloc.c

```diff
@@ -229,6 +235,15 @@ static void alloc_response_cb (flux_t *h, flux_msg_handler_t *mh,
                 flux_log_error (ctx->h,
                                 "%s: event_batch_pub_annotations: id=%ju",
                                 __FUNCTION__, (uintmax_t)id);
+            if (event_job_post_pack (ctx->event,
+                                     job,
+                                     "annotations",
+                                     EVENT_INFO_ONLY,
+                                     "{s:O?}",
+                                     "annotations", job->annotations) < 0)
+                flux_log_error (ctx->h,
+                                "%s: event_job_post_pack: id=%ju",
+                                __FUNCTION__, (uintmax_t)id);
         }
```

Comment:
If we're going to log every `event_job_post_pack()` failure in the same way, would it make sense just to log them inside the function?  (Just an idle thought  maybe for future cleanup)

Reply:
hmmm, the primary reason I do all of that logging is b/c any failures related to annotations shouldn't lead to a teardown of the job-manager, while other `event_job_post_pack()` errors should fail out with a teardown.

I guess that difference wasn't as noticeable before when we used `event_batch_pub_annotations()`.  Yeah, perhaps cleanup not necessary here, but to keep in mind for later.